src/components/FormComponents/MultiPicker.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import _ from 'lodash';
+
+// The value is extended so that at the end there is always an element that is null or undefined.
+// NOTE: The elements that are empty also include empty strings
+export default function MultiPicker(props) {
+  let {value, onUpdate, component} = props;
+  if (!_.isArray(value)) {
+    return <div></div>;
+  }
+  let lastItem = value[value.length - 1];
+  let extendedValue = value.slice();
```

Comment:
It's not clear from the code what that `extendedValue` represents.  Could you add some code documentation that explains what is happening in this function?